components/slider-carousel/index.ts

```diff
@@ -0,0 +1,4 @@
+export {default as ReuseSliderCarousel} from "./src/ReuseSliderCarousel"
+export {default as HeadlessSliderCarousel} from "./src/HeadlessSliderCarousel"
+export {default as Slider} from "./src/Slider"
```

Comment:
What are this **Slider** and **Progress** which are exposed here?

Reply:
slider renamed with ReuseSlider which has feature of slider-carousel with Progress bar.
and Progress is removed from there because no need f progress it is internally used in ReuseSlider.